src/effects/nyquist/Nyquist.cpp

```diff
@@ -1695,12 +1695,8 @@ bool NyquistEffect::ProcessOne(
    if (outChannels < static_cast<int>(mCurNumChannels)) {
       // Be careful to do this before duplication
       out->Flush();
-      auto dup = (*out->Duplicate()->begin())->SharedPointer<WaveTrack>();
-      // Must destroy one temporary list before repopulating another with
-      // correct channel grouping
+      tempList = out->MonoToStereo();
       nyxContext.mOutputTracks.reset();
-      tempList = TrackList::Temporary(nullptr, out, dup);
-      assert(!dup->IsLeader());
```

Comment:
Move the `MonoToStereo` call after the `reset`, consisntently with the comment.

Did you hit a breakpoint here in the debugger?  I think you need to do what I wrote to avoid putting the `TrackList` objects into an inconsistent state.